### ISSUE TITLE
Gphoto2 don't take pictures the next morning.

### ISSUE THREAD

**Issue by Spirit328** [gphoto/gphoto2#1]
Hi guys!
First of all, let me say a huge, major, incredible big THANK YOU for gphoto2. 
What a great tool. :+1:  :+1:  :+1:  :+1:  :+1:  :+1:  :+1:  :+1:  :+1:  :+1: 

I use it with my Raspi B with the newes Raspian and it works fine. Well, almost.
So I aimed for something bigger, than just taking some pictures in a row. I wanted to take some fast motion of the rising sun every morning.
So I just created a cronjob, tested it, changed it to a point in time for the next morning and went to sleep.
The next morning, there were no pictures at all. :-1: 
Not what you may guess, I switched off the power down option of my Canon EOS 450 D!
Defently positiv! And it still was switched off the next morning and the camera was still alive. (Accu: 81%)

Let's try something simple. 
```
gphoto2 --list-config
```
NO response from the raspi. I switched the camera off and on and did it again. The usual printout appeared on the screen. Hmmmm.

I changed the cronjob to about an hour into the future. - Worked!
I changed the cronjob to three hours - Did not work.

Any ideas?

Cheers
Thorsten

**Comment by msmeissn** [comment]
Was the camera powered down in the meantime? The camera will switch off after a while?

even if you swithc it off? can you observice it without computer?

**Comment by maglub** [comment]
I don't think this is an issue with gphoto2, but rather what the camera is doing after long periods of inactivity. I am also using a Raspberry Pi to control my camera, and I have more or less the same issue with my Canon 7D after long periods of inactivity. After searching the internet, I found two solution:

* Build an adapter to the shutter release port like here: http://www.davidhunt.ie/motorised-time-lapse-rail-with-raspberry-pi/3/

By using the "focus pin" you can wake up your camera.

* Simple McGyver hack, using a small piece of rubber/plastic/anything that you can tape onto the trigger button so it goes in half way. You will have to turn of "continuous focus" for this not to drain your battery. As long as the button is half way in, your camera will stay awake.

**Comment by Spirit328** [comment]
Sorry guys for not replying in so long. I had had a two bereavement in my family shortly after each other.

Thank you very much "maglub". I will give it a try as soon I have a little more freetime. 
I like the electronic-way better than the "McGyver" hack, although it has its own charme. Maybe it is a good alternative, if one has forgotten the "special-lead". ;-)

I will reply as soon as I have more news to share!

Nevertheless: Gphoto2 is :+1:

**Comment by msmeissn** [comment]
Just checking back after quite sometime to see if it was resolved ;)
